02-JS-Intro-jQuery/practice.js

```diff
@@ -0,0 +1,74 @@
+$(window).ready(function () {
+    $("section").fadeIn(3000);
+    $(".alias").focus();
+    $("#yourname").click(function () {
+        var nombre = "Juan";
+        $.ajax({
+            url: "http://bootcamp.aws.af.cm/welcome/" + nombre,
```

Comment:
It would be nice to bind "nombre" to an input's value rather than a string (line#5).